ethereum_history_api/circuits/lib/README.md

```diff
@@ -1,58 +1,78 @@
-# Noir Ethereum history api circuits
+# Noir Ethereum history api library
 
-Noir circuits to generate account and storage proofs for historical blocks.
+Noir library for proving and verifying historical data on the Ethereum blockchain and other EVMs.
 
-## Compilation
+**Disclaimer:** This page is supposed to give you an idea about the usage & structure of the library, but it's still a good idea to consult the code for exact definitions of functions & types.
```

Comment:
& -> and

Reply:
I would still add a section about generic types at the bottom.
How to pick parameters and so on.
We can do it in next PR, but that is important

Reply:
We should add separate section on developers responsibility to check block hash

Reply:
Added to notion as a TODO for the next PR. This will be a lot of copy pasting of types. They will now live in three places. Notion, Code & docs